src/app/templates/categories/category-list/category-list.component.html

```
@@ -27,7 +28,11 @@ <h2 class="title-medium">Loading ...</h2>
   } @else if (templateCategoryGroupsQuery.isSuccess()) {
     @if (templateCategoryGroupsQuery.data().length === 0) {
       <p>No categories found</p>
-      <button mat-flat-button (click)="openCategoryCreationDialog()">
+      <button
+        mat-flat-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)